tools/block-generator/run_tests.sh

```diff
@@ -17,7 +17,7 @@ help() {
   echo " -r|--report-dir directory where the report should be written."
   echo " -d|--duration   test duration."
   echo " -l|--level      log level to pass to Indexer."
-  echo " -g|--generator  use a different indexer binary to run the generator."
```

Comment:
Since it's no longer part of the indexer binary, this should be required. Mark as required on line 79

Reply:
updated.